components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/services/InputValidationManagementService.java

```diff
@@ -75,4 +76,14 @@ List<ValidationConfiguration> getInputValidationConfiguration (String tenantDoma
      * @return configuration.
      */
     List<ValidationConfiguration> getConfigurationFromUserStore(String tenantDomain) throws InputValidationMgtException;
+
+    /**
+     * Check whether provided validator combination is applicable to the field.
+     *
+     * @param field Field validators are configuring.
```

Comment:
Give some examples of the fields we use. I hope the fields are attributes like username and password. If so the description for the param is not matching